javalanglib/src/main/scala/java/lang/Throwables.scala

```diff
@@ -119,7 +119,7 @@ class Throwable protected (s: String, private var e: Throwable,
       val thisLength = thisTrace.length
       val parentLength = parentTrace.length
 
-      sprintln("Caused by: " + wCause.toString)
+      sprintln(s"Caused by: ${wCause}")
```

Comment:
Any reason for the braces?

Reply:
Hum, no reason. I removed them. Good catch.